### DISCUSSION TITLE
将需要的任意内容生成阅读模式并加在稍后读查看

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#3216] [使用技巧]
> 将页面的任意内容（非全部正文）加入到稍后读并查看。

https://user-images.githubusercontent.com/81074/146664541-2474fd73-1155-4130-bae9-8064bc77cd38.mp4

## 使用场景

并不是每个页面的内容都需要生成阅读模式，比如简悦用户也是我的好友 [Shyrism](https://t.me/shyrism) 的这篇 [Newsletter](https://shyrz.zhubai.love/posts/2083579225017053184) ，其中 **东京部分** 的内容我非常喜欢，就可以使用此方式生成阅读模式，并在稍后读中直接查看（这部分内容，而非全文）

## 使用方式

> 上图视频就是操作方式，很简单，用鼠标选中需要的内容（无论是文字、图片、还是其它）然后右键 → 生成阅读模式即可。

![2021-12-19_13-10-28](https://user-images.githubusercontent.com/81074/146664667-e84afa7a-cb58-41c7-b661-aa5a8f71eacf.png)

## 使用前提

1. 开启右键 **将选中内容生成阅读模式**

![image](https://user-images.githubusercontent.com/81074/146664697-bebbb7b3-0760-42df-8aee-f3822e9e2240.png)

2. 配置 [知识库](https://github.com/Kenshin/simpread/discussions/2221)

## 引申

1. 如果还需要在外网访问的话，则配置 [资料库](https://github.com/Kenshin/simpread/discussions/2956) 即可。

2. 在生成的阅读模式里面可以使用 [清除任意元素](http://ksria.com/simpread/docs/#/%E9%9A%90%E8%97%8F%E4%BB%BB%E6%84%8F%E5%85%83%E7%B4%A0) 来清理生成后的任意内容

https://user-images.githubusercontent.com/81074/146664871-b30defa7-b2c1-4f69-9fa6-cc126db66896.mp4

3. 修改名字

> 修改稍后读的名字后再导出 HTML，让这个稍后读的标题更符合内容。

https://user-images.githubusercontent.com/81074/146664990-67840228-defd-48a0-892d-d6a10141b03f.mp4

**注意**

> 当修改标题再生成 HTML 后，**无法自动删除原标题对应的 HTML**，需手动（在 `output` 里）删除。

## 关联

1. [右键菜单](http://ksria.com/simpread/docs/#/右键菜单)
2. [隐藏元素](http://ksria.com/simpread/docs/#/隐藏任意元素)
3. [知识库](https://github.com/Kenshin/simpread/discussions/2221)
4. [资料库](https://github.com/Kenshin/simpread/discussions/2956)

**Comment by yomizzz** [comment]
好神奇，我这样加入稍后读后，还是全文，不是我选中的内容…… 文章地址是 https://sspai.com/post/70525

**Reply by Kenshin** [reply]
应该是没配置知识库，教程里面有提。

**Reply by yomizzz** [reply]
刚才在稍后读里打开发现又不是全文了，是我精简保存的，好神奇🤣

**Reply by Kenshin** [reply]
哈哈，没问题就好，再多观察下。

也有可能是：调用本地失败而后又读取的原文。（简悦确实有这个机制，但读取本地失败的情况太少见了...或者说应该不存在。
